.github/workflows/e2e_detox_mapping.yml

```diff
@@ -43,18 +43,24 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
         with:
-          ref: ${{ github.ref }}
+          #ref: ${{ github.ref }}
```

Comment:
This can be removed if it's no longer needed. Unless it was accidentally commented out